docs/app/components/migrate-link/migrate-link.component.html

```diff
@@ -0,0 +1,43 @@
+<ng-template #popTemplate>
+    <table class="migrate-popover">
+        <tbody>
+            <tr>
+                <th colspan="2"><a [attr.href]="'https://pages.github.houston.softwaregrp.net/Inner-Source-Incubating/ux-aspects-universal/angular-storybook/?path=/docs/' + migration.href">UX Aspects Universal: {{ migration.title }}</a></th>
```

Comment:
I think it would be good if the base URL for the Universal storybook was stored centrally, check the plunker URL for an example.